lib/roger_ui/queues.ex

```diff
@@ -1,18 +1,54 @@
-defmodule RogerUi.Queues do
+defmodule RogerUI.Queues do
   @moduledoc """
-  Normalize queues from nodes
+  Normalizes nodes data structures from Roger.Info.running_jobs() function in order to obtain queues:
+
+  Given a nested data structure, where each element contents nested items:
+    input = [
```

Comment:
Can you put code examples in markdown code blocks?

Reply:
done